sm64-tas-scripting/Inputs.cpp

```diff
@@ -283,10 +293,9 @@ int M64::load()
 	size_t err;
 
 	try {
-		if ((err = fopen_s(&f, fileName, "rb")) != 0) {
-			std::cerr << "Bad open of file " << fileName << " Error: " << err << std::endl;
-			exit(EXIT_FAILURE);
-		}
+    if ((f = fopen(fileName.c_str(), "rb")) == nullptr) {
```

Comment:
MSVC doesn't like this: 

'FILE *fopen(const char *,const char *)': cannot convert argument 1 from 'const std::filesystem::path::value_type *' to 'const char *'